cmd/edit.go

```
@@ -120,6 +136,70 @@ func init() {
 	}
 }
 
+func editSetMetaData(userSet *set.Set, metaData string, reason transfer.Reason, reviewDate, removalDate string) {
+	if reason == transfer.Unset && reviewDate == "" && removalDate == "" && metaData == "" {
+		return
+	}
+
+	existingMeta := userSet.Metadata
+
+	reason = givenOrExistingReason(reason, existingMeta)
+	reviewDate = givenOrExistingReviewDate(reviewDate, existingMeta)
+	removalDate = givenOrExistingRemovalDate(removalDate, existingMeta)
+	metaData = givenOrExistingMetaData(metaData, userSet)
+
+	meta, err := transfer.HandleMeta(metaData, reason, reviewDate, removalDate, existingMeta)
+	if err != nil {
+		die(err)
```

I think it's better to return an error and propagate it to `RunE` so it can process all errors in a consistent way (just return to cobra). The same for all `die`s bellow.

---

Now returning the errors, changed it for all the `die`s